ripple/src/main/webapp/businessdetails.js

```diff
@@ -1,7 +1,63 @@
 /* When user clicks on a team member, add highlight to that avatar */
-$(document).ready(function(){
+$(document).ready(() => {
   $(".bd-avatar").click(function() {
```

Comment:
Can this be an inline function?

Reply:
Fixed document.ready to inline function syntax. Nested function responding to click cannot be an inline function